udata_front/theme/gouvfr/assets/js/components/spam/MarkAsNoSpam.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <button
+    class="fr-btn fr-btn--sm fr-btn--icon-right fr-icon-feedback-fill"
+    @click="markAsNoSpam"
+    v-if="show"
+  >
+    {{$t('Mark as no spam')}}
+  </button>
+</template>
+<script>
```

Comment:
Can you convert the component to `<script setup lang="ts">` like the [CardMD](udata_front/theme/gouvfr/assets/js/components/dataset/CardMDList.vue) ? This is the current syntax used.

Reply:
Done thx